### PR TITLE
Fix React Errors & Improve About/Take Action Styling

### PR DESCRIPTION
**Pull request by vkoves**
## Overview

Made a few styling tweaks, particularly:

- Made headings bold by default
- Added some spacing to `h1` and `h2` elements

Also cleaned up several React errors where attributes where not camel case (e.g. `frameborder` needs to be `frameBorder` and `class` needs to be `className`). 

| Before | After |
| --------- | ------ |
| decarbmystate com_about (1)  | localhost_8000_take-action |
| decarbmystate com_about | localhost_8000_illinois_ |

Also did some refactors resulting from this, removing extra `font-weight-bold` classes that are now no longer needed.

### Demo

See description.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Confirm the homepage and state details page are unchanged
* Confirm Take Action and About page look better and are more readable

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -41,10 +41,9 @@ const IndexPage = ({data}) => {
         description="You can help decarbonize your state, and it's a lot simpler than you think! Learn how with Decarbonize My State."
       />
       <div className="main-header">
-        <h1 id="main" className='display-4 text-center font-weight-bold'>
-          What does take...</h1>
-        <h1 className='display-4 text-center font-weight-bold'>
-          to decarb your state?
+        <h1 id="main" className="display-4 text-center font-weight-bold">
```
Comment (vkoves) [inline src/pages/index.js:44]: We had two `h1` elements, which doesn't make sense. In terms of semantics, that would mean we are saying the page has two titles.

File: src/pages/index.js [hunk 2]
```diff
@@ -41,10 +41,9 @@ const IndexPage = ({data}) => {
...
+          What does take...<br/>
+          to decarbonize your state?
```
Comment (vkoves) [inline src/pages/index.js:46]: Changed decarb to decarbonize in this context for clarity. Decarb is a title short-hand but in a sentence the full word is useful.

File: src/pages/index.js [hunk 3]
```diff
@@ -77,7 +76,7 @@ const IndexPage = ({data}) => {
         <p className="h4 mt-4">
           Learn how to <strong>electrify your own machines</strong> and <strong>pass local policy</strong> to electrify the rest
         </p>
-        <Link className='btn btn-lg btn-success mt-4' to='/take-action'>Take Action</Link>
```
Comment (vkoves) [inline src/pages/index.js:80]: Also cleaned up using single quotes for attributes. I'm surprised the linting doesn't weight in here 🤷🏻‍♂️

**Review by herrerarobert — DISMISSED** [review]
Looks good

**Review by derekeder — APPROVED** [review]
love it. much improved for readability

**Pull request merged**